services/app-api/src/emailer/templateHelpers.ts

```diff
@@ -130,6 +126,21 @@ const generateStateReceiverEmails = (
     return pruneDuplicateEmails(stateReceiverEmails)
 }
 
+//Finds all programs ids in a package and combines them into one array removing duplicates.
+const findAllPackageProgramIds = (
+    pkg: UnlockedHealthPlanFormDataType | LockedHealthPlanFormDataType
+): string[] => {
+    const programs = [...pkg.programIDs]
```

Comment:
This could be a good place to use a `Set` instead of an array. then you dont have to do any of the “if .includes()” checks yourself.

Reply:
Done! 

I thought `Set()` was added in es6 and not available in es5 for some reason. In our `tsconfig.json` in `app-api` the `complierOptions` has target set to es5.

Reply:
looks like you backed out the Set changes, I thought it did work?

I was imagining it more like this:
```
  const programs = new Set(pkg.programIDs)
  if (pkg.submissionType === 'CONTRACT_AND_RATES' && pkg.rateProgramIDs) {
     pkg.rateProgramIDs.forEach( id => programs.add(id))
  }
  return programs
```

I don't think there's anything wrong with returning a set instead of an array, all the iterators will still work.